services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -48,45 +48,103 @@ import {
 import DocumentTable from "@mds/common/components/documents/DocumentTable";
 import { MineDocument } from "@mds/common/models/documents/document";
 import { Link } from "react-router-dom";
+import {
+  PROJECT_SUMMARY_DOCUMENT_TYPE_CODE_STATE,
+  PROJECT_SUMMARY_DOCUMENT_TYPE_CODE,
+} from "../..";
 
 const RenderEMAPermitCommonSections = ({ code, isAmendment, index }) => {
   const dispatch = useDispatch();
   const purposeLabel = isAmendment
     ? "Additional Amendment Request Information"
     : "Purpose of Application";
-
+  const authType = isAmendment ? "AMENDMENT" : "NEW";
   const { authorizations, mine_guid, project_guid, project_summary_guid } = useSelector(
     getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY)
   );
-
   const codeAuthorizations = authorizations[code];
   const { AMENDMENT, NEW } = codeAuthorizations;
   const sectionValues = isAmendment ? AMENDMENT[index] : NEW[index];
-  const dfaRequired =
-    isAmendment &&
-    sectionValues?.amendment_changes?.some((val) => ["ILT", "IGT", "DDL"].includes(val));
-  const [showDocSection, setShowDocSection] = useState(sectionValues?.exemption_requested || false);
+  const [showExemptionSection, setshowExemptionSection] = useState(
+    sectionValues?.exemption_requested || false
+  );
 
   const projectSummaryDocumentTypesHash = useSelector(getProjectSummaryDocumentTypesHash);
 
   const onChange = (value, _newVal, _prevVal, _fieldName) => {
-    setShowDocSection(value);
+    setshowExemptionSection(value);
+  };
+
+  const isDocumentTypeRequired = (type) => {
```

Comment:
I'd be inclined to move this into the `AuthorizationSupportDocumentUpload` component instead of passing all those props, but that's more of a style thing.

Reply:
The `isDocumentTypeRequired` function has been moved to `AuthorizationSupportDocumentUpload` component now.